config/locales/es.yml

```diff
@@ -0,0 +1,25 @@
+es:
+  devise:
+    omniauth_callbacks:
+      success: "Sesión realizada con tu cuenta de %{kind}."
+  spree:
+    user_was_not_valid: "El usuario no es válido."
+    add_another_service: 'Añadir otro servicio para inicio de sesión:'
+    authentications:
+      destroy: 'Método de autenticación eliminado satisfactoriamente.'
+    back_to_authentication_methods_list: "Volver a la lista de métodos de autenticación"
+    edit_social_method: 'Editando metodo de autenticación'
+    new_social_method: 'Nuevo método de autenticación'
+    no_authentication_methods_found: "No se encuentran ningún método de autenticación"
+    one_more_step: 'Un paso más para completar tu registro desde %{kind}'
+    remove_authentication_option_confirmation: '¿Está seguro que quiere eliminar este método de autenticación?'
+    sign_into_account: 'Puedes iniciar sesión usando:'
+    sign_in_through_one_of_these_services: 'Iniciar sesión a través de uno de estos servicios:'
+    social_api_key: 'API Clave'
```

Comment:
`API Clave` -> `Llave del API` or maybe we can use `Clave del API` :thinking:

Reply:
or 'Clave publica de la API'

Reply:
I like your suggestion better :ok_hand: